src/DonutScatter.js

```diff
@@ -88,7 +88,7 @@ var DonutScatter = function() {
 
             // Calculate x and y scales
             let xMax = d3.max(data, (d) => +d[xAccessor]) * 1.01;
-            let xMin = d3.min(data, (d) => +d[xAccessor]) * .5;
+            let xMin = d3.min(data, (d) => +d[xAccessor]) -1000;
```

Comment:
This change shouldn't be here. Will probably work itself out when you reverse-merge from master.